eth-bytecode-db/eth-bytecode-db/src/bin/database_populate.rs

```diff
@@ -0,0 +1,70 @@
+use entity::sources;
```

Comment:
why not make this a benchmark? may be use criterion

Reply:
Maybe, but inserting 500.000 contracts still takes a lot of time. Even 10.000 takes about 30 seconds

Reply:
you can write preparation code that will not count inside benchmark, and you can make it configurable with env variable (so you'll insert the contracts only once) :)

Reply:
I will add this point about benchmarking to main issue